File: Code/Java-code/Person.java
abstract class Person {
	//public abstract void run();

	private String name;
	private JobType job;
	private int phoneNumber;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public void setJob(JobType job) {
		this.job = job;
	}

	public JobType getJob() {
		return this.job;
	}

	public int getPhoneNumber() {
		return this.phoneNumber;
	}

	public void setPhoneNumber(int phoneNumber) {
		this.phoneNumber = phoneNumber;
	}
}

File: Code/PHP-Code/src/Person.php
<?php
abstract class Person{
    private String $name;
    private int $phoneNumber;

    public function getName(): string
    {
        return $this->name;
    }

    public function getPhoneNumber(): int
    {
        return $this->phoneNumber;
    }

    public function setName(string $name): void
    {
        $this->name = $name;
    }

    public function setPhoneNumber(int $phoneNumber): void
    {
        $this->phoneNumber = $phoneNumber;
    }


}